sdk/nodejs/runtime/closure/parseFunction.ts

```diff
@@ -440,6 +440,15 @@ function computeCapturedVariableNames(file: ts.SourceFile): CapturedVariables {
     return result;
 
     function isBuiltIn(ident: string): boolean {
+        // The __awaiter is never considered built-in.  We do this as async/await code will generate
+        // this (so we will need it), but some libraries (like tslib) will add this to the 'global'
+        // object.  If we think this is built-in, we won't serialize it, and the function may not
+        // actually be available if the import that caused it to get attached isn't included in the
+        // final serialized code.
+        if (ident === "__awaiter") {
+            return false;
+        }
```

Comment:
while i'm not a fan of this sort of hardcoding, it's really the simplest and safest thing i can think of doing here.